README.rst

```diff
@@ -17,6 +17,40 @@ For more details `mirrormanager <https://fedorahosted.org/mirrormanager/>`_
 Hacking
 -------
 
+Hacking with Vagrant
+~~~~~~~~~~~~~~~~~~~~
+Quickly start hacking on mirrormanager2 using the vagrant setup that is included
+in the repo is super simple.
+
+First, make a copy of the Vagrantfile example::
+
+    $ cp Vagrantfile.example Vagrantfile
```

Comment:
If we don't edit the vagrantfile, what is the advantage of not shipping it as ``vagrantfile``?